Sources/PrimerSDK/Classes/Data Models/PaymentMethodConfigurationType.swift

```diff
@@ -231,6 +243,7 @@ public enum PaymentMethodConfigType: Codable, Equatable {
         case adyenDotPay
         case adyenGiropay
         case adyenIDeal
+        case adyenInterac
```

Comment:
How come and adyenPayTrail is not needed?

Reply:
```
        case adyenInterac
        case adyenPayTrail
```